.github/workflows/release.yaml

```diff
@@ -0,0 +1,67 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      repository:
+        description: Repository (in owner/repo format)
+        required: true
+        default: kiali/kiali.io
+        type: string
+      releasing_branch:
+        description: Branch to release
+        required: true
+        default: staging
+        type: string
+
+jobs:
+  initialize:
+    name: Initialize
+    runs-on: ubuntu-20.04
+    outputs:
+      releasing_version: ${{ steps.releasing_version.outputs.releasing_version }}      
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ github.event.inputs.repository }}
+          ref: ${{ github.event.inputs.releasing_branch }}      
+      
+      - name: Determine releasing version        
+        id: releasing_version
+        run: |
+          CURRENT_VERSION=$(awk '/## NEXT/,/url/' config.toml | grep -o '\"v.*\"' | tr -d '\"' | xargs)
```

Comment:
Does it really works with the `xargs`? I fail to understand why the `xargs` is needed :confused: .

Reply:
I'm using it to trim the version.

Reply:
OK, understood